kubernetes/samples/scripts/create-rcu-schema/common/createRepository.sh

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# http://oss.oracle.com/licenses/upl.
+#
+
+. /u01/oracle/wlserver/server/bin/setWLSEnv.sh
+
+echo "Check if the DB Service is Ready to accept Connection ?"
+connectString=${1:-oracle-db.default.svc.cluster.local:1521/devpdb.k8s}
+schemaPrefix=${2:-domain1}
+echo "DB Connection URL [$connectString] and schemaPrefix is [${schemaPrefix}]"
+
+max=20
+counter=0
+while [ $counter -le ${max} ]
+do
+ #java utils.dbping ORACLE_THIN "sys as sysdba" Oradoc_db1 ${connectString}
+ java utils.dbping ORACLE_THIN scott tiger ${connectString} > dbping.err 2>&1
```

Comment:
this wont work unless the db has the sample schemas installed
nc server:port would be a safer test